docs/cli.md

```diff
@@ -64,54 +68,60 @@ The general structure for running commands with the Gravitino CLI is `gcli entit
  -x,--index              display index information
  -z,--provider <arg>     provider one of hadoop, hive, mysql, postgres,
                          iceberg, kafka
- ```
+```
 
-## Commands
+## Commands Types
 
 The following commands are used for entity management:
 
-- list: List available entities
-- details: Show detailed information about an entity
-- create: Create a new entity
-- delete: Delete an existing entity
-- update: Update an existing entity
-- set: Set a property on an entity
-- remove: Remove a property from an entity
-- properties: Display an entities properties
+- `create`: Create a new entity
+- `delete`: Delete an existing entity
+- `details`: Show detailed information about an entity
+- `list`: List available entities
+- `properties`: Display an entity's properties
+- `remove`: Remove a property from an entity
+- `set`: Set a property on an entity
+- `update`: Update an existing entity
 
 ### Setting the Metalake name
 
-As dealing with one Metalake is a typical scenario, you can set the Metalake name in several ways so it doesn't need to be passed on the command line.
+Considering that managing a metalake is a typical scenario, you can set the metalake
+name in several ways so it doesn't need to be passed on the command line.
 
-1. Passed in on the command line via the `--metalake` parameter.
-2. Set via the `GRAVITINO_METALAKE` environment variable.
-3. Stored in the Gravitino CLI configuration file.
+1. Specify the `--metalake` argument on the command line.
+1. Set the `GRAVITINO_METALAKE` environment variable.
+1. Save it in the Gravitino [CLI configuration file](#config-file).
```

Comment:
This change (Save it) doesn't work well.

Reply:
Yup. Good suggestion.